opentelemetry-proto/src/proto.rs

```diff
@@ -182,6 +189,17 @@ pub(crate) mod serializers {
         let s: String = Deserialize::deserialize(deserializer)?;
         s.parse::<i64>().map_err(de::Error::custom)
     }
+
+    pub fn serialize_vec_u8_as_base64_string<S: Serializer>(v: &Vec<u8>, s: S) -> Result<S::Ok, S::Error> {
```

Comment:
Are we planning to use these methods in the future, given that they are currently unused?

Reply:
Yes but I'm not sure where or how. I'm having some trouble grokking the codebase because of all of the codegen.

Reply:
I don't see bytes <-> base64 conversion required outside of `AnyValue::Value`. The only other candidates could have been `traceid`, `spanid` byte arrays but they are encoded as hex-str as per the [specs](https://github.com/open-telemetry/opentelemetry-proto/blob/v1.5.0/docs/specification.md#json-protobuf-encoding). So it should be safe to remove these methods for now, and bring back later if required.

Reply:
ok removed